### ISSUE TITLE
Error: [CRITICAL]: Could not tick level "worldname": Undefined variable: block - since build 289

### ISSUE THREAD

**Issue by ecoron** [ImagicalCorp/ImagicalMine#509]
since build 289 (and still exists in 299) i find this error in my server.log:
```
[CRITICAL]: Could not tick level "worldname": Undefined variable: block
```

i think it must be something similar like the fixed issue #493

**Comment by PocketKiller** [comment]
is this the full message? it should contain in blabla/blabla.php at line 293929293

**Comment by ecoron** [comment]
it's the full message

**Comment by matcracker** [comment]
We need more information, can you enable the debug level to 2 in the config?

**Comment by robske110** [comment]
(Pocketmine.yml)

**Comment by ecoron** [comment]
yes i will try.
i searched also over the code for usage of $block (its a lot 800 times) but didn't find anything wrong yet

**Comment by thebigsmileXD** [comment]
😂 welll.. I know where.. I'll check later

**Comment by ecoron** [comment]
after change to build 302 yesterday, the error is gone, seems to be solved already.

**Comment by thebigsmileXD** [comment]
Haven't i closed this already? Because i fixed it

**Comment by PocketKiller** [comment]
@thebigsmileXD you didn't, if you want to close it by a commit do like Fixed #509 or closes #509

**Comment by thebigsmileXD** [comment]
I did in.. Idk, on phone now. But there was another issue with the same content.
